ladybug_vtk/to_vtk.py

```diff
@@ -0,0 +1,40 @@
+"""Functions to create VTK objects from Ladybug Objects."""
+
+import vtk
+from typing import List
+from ladybug_geometry.geometry3d import Point3D, Polyline3D
+from .polydata import PolyData
+
+
+def create_points(points: List[Point3D]) -> PolyData:
```

Comment:
We should probably name the functions similar to other translation libraries: https://github.com/ladybug-tools/ladybug-rhino/blob/master/ladybug_rhino/togeometry.py

```suggestion
def to_vtk_points(points: List[Point3D]) -> PolyData:
```